src/OrbitGl/GraphTrack.cpp

```diff
@@ -37,24 +37,27 @@ GraphTrack<Dimension>::GraphTrack(CaptureViewElement* parent,
     : Track(parent, timeline_info, viewport, layout, module_manager, capture_data),
       series_{series_names, series_value_decimal_digits, std::move(series_value_units)} {}
 
+template <size_t Dimension>
+bool GraphTrack<Dimension>::HasLegend() const {
+  return !IsCollapsed() && Dimension > 1;
+}
+
 template <size_t Dimension>
 float GraphTrack<Dimension>::GetHeight() const {
-  // Top content margin is counted twice because it it inserted before and after the legend
-  float height = layout_->GetTrackTabHeight() + layout_->GetTrackContentTopMargin() * 2 +
-                 GetLegendHeight() + GetGraphContentHeight() +
-                 layout_->GetTrackContentBottomMargin();
-  return height;
+  // Top content margin is counted twice when there are legends because it is inserted above and
```

Comment:
The content margin MIGHT be counted twice....

Reply:
No might here. When there are legends, it MUST be counted twice.

Reply:
When there are legends yes, but not always. Again, I would prefer to use "might".

Reply:
Wow, I've just seen it. Please ignore my comment